code/workspaces/web-app/src/components/common/ConfirmDialog.js

```diff
@@ -24,12 +24,12 @@ function EditProjectDialog({ state, onSubmit, title, body, onCancel }) {
   );
 }
 
-EditProjectDialog.propTypes = {
+ConfirmDialog.propTypes = {
   onSubmit: PropTypes.func.isRequired,
   onCancel: PropTypes.func.isRequired,
   state: PropTypes.object.isRequired,
   title: PropTypes.string.isRequired,
   body: PropTypes.string.isRequired,
 };
 
-export default EditProjectDialog;
+export default ConfirmDialog;
```

Comment:
Actually, all I did was use the existing EditProjectDialog and move it into common/ConfirmDialog.js, so it's used elsewhere.  Happy to change it for both if you want?

Reply:
I think the place you reused it from might be the only other place it is like this so I think it would be worth updating to match the style of the second screenshot.

Reply:
Done